lib/vintage_net_qmi/mcc_mnc.ex

```diff
@@ -5,10 +5,14 @@ defmodule VintageNetQMI.MCCMNC do
   This database uses data from https://s3.amazonaws.com/mcc-mnc.net/mcc-mnc.csv.
   """
 
+  @external_file "lib/mcc-mnc.csv"
+  # convert the database file into something smaller
+  :ok = VintageNetQMI.MixProject.build_csv(@external_file)
```

Comment:
Oh, now I understand the feature that you were talking about with `@external_file`. That's the feature that makes this `.ex` file depend on `lib/mcc-mnc.csv`. I really think there's another way of doing this that's solely in `mix.exs`, but if I don't find it quickly, I agree that this will only process the `.csv` file when it changes.